internal/langserver/handlers/complete.go

```diff
@@ -47,6 +47,13 @@ func (h *logHandler) TextDocumentComplete(ctx context.Context, params lsp.Comple
 	}
 	d.SetSchema(schema)
 
+	expFeatures, err := lsctx.ExperimentalFeatures(ctx)
+	if err != nil {
+		d.PrefillRequiredAttributes = false
```

Comment:
You may know this already, but in Go you can safely assume that any `bool` struct field always defaults to `false`, so this has basically no effect.

Also the condition under which this errors out is pretty rare, but when it does it almost certainly points to a bug which will need to be fixed, so I'd return the error here, just like we do it in the `DidSave` handler:
https://github.com/hashicorp/terraform-ls/blob/8840e7ba573d1e431898779b80a9ba2d464ce74d/internal/langserver/handlers/did_save.go#L14-L17